projects/storefrontlib/shared/components/card/card.component.html

```diff
@@ -51,14 +55,58 @@
             ></cx-truncate-text-popover>
           </ng-template>
         </div>
-        <div class="cx-card-paragraph" *ngFor="let item of content.paragraphs">
-          <div class="cx-card-paragraph-title">{{ item.title }}</div>
-          <div *ngFor="let text of item.text">
-            <div class="cx-card-paragraph-text">
-              {{ text }}
+        <form [formGroup]="saveForm">
+          <div
+            class="cx-card-paragraph"
+            *ngFor="let item of content.paragraphs; trackBy: trackByFn"
+          >
+            <div class="cx-card-paragraph-title">{{ item.title }}</div>
+            <div
+              *ngFor="let text of item.text; let i = index; trackBy: trackByFn"
```

Comment:
```suggestion
              *ngFor="let text of item.text; let i = index; trackBy: trackByIndex"
```
lets rename the track by function to trackByIndex